pkg/validator/validators.go

```diff
@@ -360,22 +391,53 @@ func validateMRCProvider(mrc *configv1alpha2.MeshRootCertificate) error {
 	return nil
 }
 
-// checkForExistingActiveMRC returns true if there are active MRCs in addition to the specified MRC
+// mrcWithIntentExist returns true if there are active MRCs in addition to the specified MRC
 // Returns false if there are no active MRCs or if the specified mrc will be the only active MRC
-func (kc *validator) checkForExistingActiveMRC(mrc *configv1alpha2.MeshRootCertificate) (bool, error) {
+func mrcWithIntentExist(mrcs []*configv1alpha2.MeshRootCertificate, mrc *configv1alpha2.MeshRootCertificate) bool {
+	for _, m := range mrcs {
+		if m.Spec.Intent == mrc.Spec.Intent && m.Name != mrc.Name {
+			return true
+		}
+	}
+
+	return false
+}
+
+func (kc *validator) getCertsInUseMRCs() ([]*configv1alpha2.MeshRootCertificate, error) {
```

Comment:
```suggestion
func (kc *validator) getUseableMRCs() ([]*configv1alpha2.MeshRootCertificate, error) {
```

I'm wary of calling MRCs "certs". That term/idea is so overloaded already in this package.